third_party/ti_simplelink_sdk/run_sdk_drivers_gen.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Texas Instruments Incorporated
+
+"""A wrapper to run the SDK makefiles for Platform driver generation
+Args:
+1. [TI Simplelink SDK Root]
+2. [Matter repository Root]
+"""
+
+import subprocess
+import sys
+
+SDK_ROOT = sys.argv[1]
+CHIP_ROOT = sys.argv[2]
+
+GCC_ARMCOMPILER_PATH = CHIP_ROOT + "/.environment/cipd/packages/arm/"
```

Comment:
.environment is not guaraneteed to be the right path. PW_ENVIRONMENT_ROOT may be set in which case this path differs (in particular CI does set a different PW_ENVIRONMENT_ROOT and this path is not valid)

Reply:
Fair point, I'm assuming _PW_ACTUAL_ENVIRONMENT_ROOT will point to the correct location in both cases?